hp_list/Makefile

```diff
@@ -1,6 +1,11 @@
+CFLAG=-D ANALYSIS_OPS
```

Comment:
`ANALYSIS_OPS` was misleading. Use `RUNTIME_STAT` which does many internal checks and maintains detailed statistics instead.

Reply:
You shall add some related comments as well.